jdaviz/app.py

```diff
@@ -1796,11 +1797,18 @@ def _reparent_subsets(self, old_parent, new_parent=None):
                     # Translate bounds through WCS if needed
                     if (self.config == "imviz" and
                             self._jdaviz_helper.plugins["Orientation"].link_type == "WCS"):
+
+                        old_angle, _, old_flip = get_compass_info(old_parent.coords, (10, 10))[-3:]
```

Comment:
Do we want to add a comment that this shape is from here?

https://github.com/spacetelescope/jdaviz/blob/0e0a3b72bf5ccb55bf8d2068fa9da10e372f6fd7/jdaviz/configs/imviz/wcs_utils.py#L455

Reply:
Or better yet, programmatically accessing the shape rather than hardcoding?

Reply:
I actually wasn't even sure if the shape matters at all, given that we only need the angle and flip. I can add a comment though,